mirrord/cli/src/container.rs

```diff
@@ -221,7 +221,11 @@ async fn create_sidecar_intproxy(
         {
             let mut container_start_command = Command::new(&runtime_binary);
 
-            container_start_command.args(["start", &sidecar_container_id]);
+            container_start_command
+                .args(["start", &sidecar_container_id])
+                .stdin(Stdio::null())
+                .stdout(Stdio::piped())
+                .stderr(Stdio::piped());
```

Comment:
Looks like we don't read these streams anywhere, so maybe you could use `Stdio::null()` instead?